publish-n4js-libs-npm.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
@dbo please review this file and file `n4js-libs/npm-publish-canary.sh`, and related files, lerna.json etc.!

Reply:
why do we want to publish to `canary` if we don't have multiple development streams? Also we don't have build promotion between the streams. Using `canary` here is pointless. Especially for everything before `1.0.0`.

Reply:
We are publishing all the npms using canary version and the dist-tag `latest`. That means, the user can/should use the latest version of the npms for now. In the future, when we introduce npm/IDE versioning, this will be improved.